components/header.tsx

```
@@ -96,44 +99,48 @@ const Header: React.FC = () => {
-            <div className={styles['action-group']}>
+          <div className={styles["nav-right"]}>
+            <div className={styles["action-group"]}>
               {/* Velt Notification Tool */}
               <VeltNotificationsTool darkMode={isDarkMode} />
             </div>
-            <div className={styles['button-group']}>
+            <div className={styles["button-group"]}>
               {/* VeltCommentsSidebar Button */}
               <VeltSidebarButton>
-                <button className={styles['ghost-btn']}>
+                <button className={styles["ghost-btn"]}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)